Feat/community post page

SEOKKAMONI opened the pull request:
## 📄 Summary
> 
커뮤니티 포스트 페이지 만듦


<br>

## 🔨 Tasks
-  


<br>

## 🙋🏻 More

gimhanul commented:
[](url)

gimhanul commented on pages/CommunityScreen.js:
```diff
@@ -1,32 +1,26 @@
+export default function CommunityScreen() {
+    // useState 로 props 보내서 페이지 체인지하는 코드 짯는데 요부분
+    // 코드 리뷰좀 부탁드려요 - 석진 -
+    const [isChoiceBtn, setChoiceBtn] = useState(true);
```
state를 true / false 말고 변수명을 `feedType`으로 두고 ['GATI', 'NORMAL'] 이런 식으로 분류해서 validation하는 게 나을 듯 하네용

arkk200 committed:
FEAT :: 프로필 페이지 제작 완료

gimhanul committed:
Merge branch 'develop' into feat/community-post-page

gimhanul committed:
REFACTOR :: toggle

- toggle 비활성화되면 background color 값 변경
- toggle componenet 추출
- 그림자 수정
- header style props 추가
